lib/utils.py

```diff
@@ -31,12 +32,24 @@ def sanitize_location(location):
     return location
 
 def get_driverpath():
-    if system() == "Windows":
+    if Os().wsl or Os().windows:
         driverpath = "./chromedriver.exe"
     else:
         driverpath = "./chromedriver"
     
     if isfile(driverpath):
         return driverpath
     else:
-        exit("The chromedriver is missing.\nPlease put it in the GHunt directory.")
\ No newline at end of file
+        exit("The chromedriver is missing.\nPlease put it in the GHunt directory.")
+
+def get_wsl_chrome_options_args():
+    chrome_options = Options()
+    if Os().wsl or Os().windows:
+        chrome_options.add_argument("--disable-gpu")
+        chrome_options.add_argument("--disable-dev-shm-usage")
+        chrome_options.add_argument("--disable-setuid-sandbox")
+        chrome_options.add_argument("--no-first-run")
+        chrome_options.add_argument("--no-zygote")
+        chrome_options.add_argument("--single-process")
+        chrome_options.add_argument("--no-sandbox")
```

Comment:
On some parts of the code, the `--no-sandbox` argument is added before `get_wsl_chrome_options_args` is called: Such cases are line 45-48 on `lib/photos.py`, and lines 64-67 on `lib/gmaps.py`.

`--no-sandbox` should always be added regardless of platform (much like the "headless" option).
I recommend you move it so it's before the `if Os().wsl or Os().windows:`

Reply:
Are you referring to `chrome_options.add_argument("--headless")`; I don't see chrome_options.add_argument("--no-sandbox") after moving it to the function in the last commit.

Reply:
Seems reasonable. I'll move `chrome_options.add_argument("--headless")` there as well.